lib/src/reactive_ble.dart

```diff
@@ -28,41 +29,22 @@ class FlutterReactiveBle {
     _trackStatus();
   }
 
-  final _methodChannel = const MethodChannel("flutter_reactive_ble_method");
-
+  /// Registry that keeps track of all BLE devices found during BLE scan.
```

Comment:
I think we shouldn't have exposed this publicly

Reply:
This registry is mutable, which allows an API consumer to mess with its contents. From this point of view, yes, it should definitely be private.